src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UpdateRfocAcceptedStatus/UpdateRfocAcceptedCommandHandler.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.Common.Misc;
+using Equinor.ProCoSys.IPO.Domain;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.ProjectAggregate;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.Certificate;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.UpdateRfocAcceptedStatus
+{
+    public class UpdateRfocAcceptedCommandHandler : IRequestHandler<UpdateRfocAcceptedCommand, Result<Unit>>
+    {
+        private readonly IInvitationRepository _invitationRepository;
+        private readonly IProjectRepository _projectRepository;
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IPlantProvider _plantProvider;
+        private readonly ICertificateApiService _certificateApiService;
+        private readonly ILogger<UpdateRfocAcceptedCommandHandler> _logger;
+
+        public UpdateRfocAcceptedCommandHandler(
+            IInvitationRepository invitationRepository,
+            IProjectRepository projectRepository,
+            IUnitOfWork unitOfWork,
+            IPlantProvider plantProvider,
+            ICertificateApiService certificateApiService,
+            ILogger<UpdateRfocAcceptedCommandHandler> logger)
+        {
+            _invitationRepository = invitationRepository;
+            _projectRepository = projectRepository;
+            _unitOfWork = unitOfWork;
+            _plantProvider = plantProvider;
+            _certificateApiService = certificateApiService;
+            _logger = logger;
+        }
+
+        public async Task<Result<Unit>> Handle(UpdateRfocAcceptedCommand request, CancellationToken cancellationToken)
+        {
+            var project = await _projectRepository.GetProjectOnlyByNameAsync(request.ProjectName);
+            if (project == null)
+            {
+                _logger.LogInformation($"Early exit in RfocAccepted handling. Project {request.ProjectName} does not exists in IPO module");
+                return new SuccessResult<Unit>(Unit.Value);
+            }
+            
+            if (project.IsClosed)
+            {
+                _logger.LogInformation($"Early exit in RfocAccepted handling. Project {request.ProjectName} is closed in IPO module");
+                return new SuccessResult<Unit>(Unit.Value);
+            }
+
+            var certificateMcPkgsModel = await _certificateApiService.TryGetCertificateMcPkgsAsync(_plantProvider.Plant, request.ProCoSysGuid);
+            var certificateCommPkgsModel = await _certificateApiService.TryGetCertificateCommPkgsAsync(_plantProvider.Plant, request.ProCoSysGuid);
+
+            if (certificateMcPkgsModel == null && certificateCommPkgsModel == null)
```

Comment:
check for certificateMcPkgsModel == null behind line 56 and certificateCommPkgsModel == null alone here (2 if-statements)